sdk/typescript/test/unit/cryptography/multisig.test.ts

```diff
@@ -27,49 +27,76 @@ describe('multisig address and combine sigs', () => {
 		let k3 = Ed25519Keypair.fromSecretKey(new Uint8Array(32).fill(0));
 		let pk3 = k3.getPublicKey();
 
-		const data = new Uint8Array([0, 0, 0, 5, 72, 101, 108, 108, 111]);
-		const digest = blake2b(data, { dkLen: 32 });
-
-		const sig1 = {
-			signature: k1.signData(digest),
-			signatureScheme: k1.getKeyScheme(),
-			pubKey: pk1,
-		};
+		const multiSigPublicKey = MultiSigPublicKey.fromPublicKeys({
+			threshold: 3,
+			publicKeys: [
+				{ publicKey: pk1, weight: 1 },
+				{
+					publicKey: pk2,
+					weight: 2,
+				},
+				{ publicKey: pk3, weight: 3 },
+			],
+		});
 
-		const ser_sig1 = toSerializedSignature(sig1);
+		const data = new Uint8Array([0, 0, 0, 5, 72, 101, 108, 108, 111]);
 
-		const sig2 = {
-			signature: k2.signData(digest),
-			signatureScheme: k2.getKeyScheme(),
-			pubKey: pk2,
-		};
+		const sig1 = await k1.signPersonalMessage(data);
+		const sig2 = await k2.signPersonalMessage(data);
+		const sig3 = await k3.signPersonalMessage(data);
 
-		const ser_sig2 = toSerializedSignature(sig2);
-		expect(
-			toMultiSigAddress(
-				[
-					{ pubKey: pk1, weight: 1 },
-					{ pubKey: pk2, weight: 2 },
-					{ pubKey: pk3, weight: 3 },
-				],
-				3,
-			),
-		).toEqual('0x37b048598ca569756146f4e8ea41666c657406db154a31f11bb5c1cbaf0b98d7');
-
-		let multisig = combinePartialSigs(
-			[ser_sig1, ser_sig2],
-			[
-				{ pubKey: pk1, weight: 1 },
-				{ pubKey: pk2, weight: 2 },
-				{ pubKey: pk3, weight: 3 },
-			],
-			3,
+		expect(multiSigPublicKey.toSuiAddress()).toEqual(
+			'0x37b048598ca569756146f4e8ea41666c657406db154a31f11bb5c1cbaf0b98d7',
 		);
+
+		let multisig = multiSigPublicKey.combinePartialSignatures([sig1.signature, sig2.signature]);
 		expect(multisig).toEqual(
-			'AwIAvlJnUP0iJFZL+QTxkKC9FHZGwCa5I4TITHS/QDQ12q1sYW6SMt2Yp3PSNzsAay0Fp2MPVohqyyA02UtdQ2RNAQGH0eLk4ifl9h1I8Uc+4QlRYfJC21dUbP8aFaaRqiM/f32TKKg/4PSsGf9lFTGwKsHJYIMkDoqKwI8Xqr+3apQzAwADAFriILSy9l6XfBLt5hV5/1FwtsIsAGFow3tefGGvAYCDAQECHRUjB8a3Kw7QQYsOcM2A5/UpW42G9XItP1IT+9I5TzYCADtqJ7zOtqQtYqOo0CpvDXNlMhV3HeJDpjrASKGLWdopAwMA',
+			'AwIANe9gJJmT5m1UvpV8Hj7nOyif76rS5Zgg1bi7VApts+KwtSc2Bg8WJ6LBfGnZKugrOqtQsk5d2Q+IMRLD4hYmBQFYlrlXc01/ZSdgwSD3eGEdm6kxwtOwAvTWdb2wNZP2Hnkgrh+indYN4s2Qd99iYCz+xsY6aT5lpOBsDZb2x9LyAwADAFriILSy9l6XfBLt5hV5/1FwtsIsAGFow3tefGGvAYCDAQECHRUjB8a3Kw7QQYsOcM2A5/UpW42G9XItP1IT+9I5TzYCADtqJ7zOtqQtYqOo0CpvDXNlMhV3HeJDpjrASKGLWdopAwMA',
 		);
 
 		let decoded = decodeMultiSig(multisig);
```

Comment:
As a follow-up we probably want to rename this method and re-home it to the multisig export

Reply:
the whole existing multisig package can be deprecated as a followup.  Rather than using `decodeMultiSig` you can just use `parseSerializedSignature`, this was left in the tests for now to ensure the new implementations matched the old implementation